### ISSUE TITLE
OnTap not firing on current week display

### ISSUE THREAD

**Issue by jovy7** [SimformSolutionsPvtLtd/flutter_calendar_view#338]
OnTap Event not firing when you are in the current week of the weekview(). Previous weeks and Future weeks are ok.

**Comment by AndreLuizNogueira** [comment]
I am also facing this issue on 1.1.0. tried 1.0.4 and it worked

**Comment by AndreLuizNogueira** [comment]
For anyone experiencing this error, it has already been fixed, just add a dependency override

```
dependency_overrides: 
  calendar_view:
    git:
      url: https://github.com/SimformSolutionsPvtLtd/flutter_calendar_view.git
```

**Comment by GiovaniHaka** [comment]
Not working for me either... even with dependency_overrides

**Comment by AndreLuizNogueira** [comment]
> Not working for me either... even with dependency_overrides

Did you tried flutter clean after the override ? For me the override solved.

**Comment by rmpt** [comment]
it works with the override, but isn't it a little iffy to add the dependency directly from the git repo? When will be this included on the release?

**Comment by AndreLuizNogueira** [comment]
> it works with the override, but isn't it a little iffy to add the dependency directly from the git repo? When will be this included on the release?

Sorry, but I have no information. I only saw that there was a commit fixing this and didn't want to revert to 1.0.4 so I tried replacing and it worked. I have no contact with the owner

**Comment by apurva010** [comment]
Closing this issue as issue got fixed in #335.
